WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Views/MigrationActionsView.swift

```diff
@@ -3,10 +3,18 @@ import WordPressUI
 
 final class MigrationActionsView: UIView {
 
-    // MARK: - Views
+    // MARK: - Properties
 
     private let configuration: MigrationActionsViewConfiguration
 
+    var style = Style.translucent {
```

Comment:
How about we keep this property private and set it in the initializer once and for all? Maybe it could also be part of its `configuration`. This way, we won't have to worry about redrawing when it gets changed, and it's probably not likely that the same instance would need to change style after being initialized, wdyt?

Reply:
The reason `style` property is mutable was to prepare for an upcoming change. In the Figma Design, there was a previous **Multi-Site Welcome Screen** design that shows the following:

- Translucent actions sheet when there is content behind the view.
- Transparent ( visual and separator both hidden ) when the user scrolls the list and no content appears behind the view.

But apparently this has changed and right now the actions sheet is always translucent whether there is content underneath or not.

I'll make the change and move the `style` property to `configuration`.

<img width="434" alt="welcome" src="https://user-images.githubusercontent.com/9609223/201186794-aa7b5c75-2f8c-430e-929d-adacc9660441.png">